src/Serilog.Sinks.Datadog.Logs/Sinks/Datadog/DatadogHttpClient.cs

```diff
@@ -116,9 +129,21 @@ private async Task Post(string payload)
                     continue;
                 }
             }
-            SelfLog.WriteLine("Could not send payload to Datadog: {0}", payload);
+            var error = new CannotSendLogEventException(payload, logEventChunk.LogEvents);
```

Comment:
nit: maybe avoid creating the obj if no _exceptionHandler set?
if(_excetpionHandler != null)
{
    var error = ...
    _exceptionHandler(error)
}

Reply:
`error` was used at line 138: https://github.com/DataDog/serilog-sinks-datadog-logs/pull/51/files/421023458406a6f339ebf0ba3eb002f141a75434#diff-68cf7ce99d677222db8ab833d4ab8ab7R138

Reply:
I meant line 138 doesn't really need the error object. It just writes a const string in this case...